msteams-platform/graph-api/rsc/grant-resource-specific-consent.md

```diff
@@ -409,6 +419,71 @@ The default value of the property `isUserPersonalScopeResourceSpecificConsentEna
 
 <br>
 
+<b> After March 14th, 2024 </b>
+
+Admins will adjust Team RSC and Chat RSC settings via PowerShell cmndlets. Below are the available states for the PowerShell settings. Each section below will show examples of how to use these states to adjust your settings.
+
+| PowerShell State | Description |
+| ---- | :----: |
+| ManagedByMicrosoft |  This is the default state for all tenants, it allows Chat RSC and Team RSC permissions to be consented for all users, but can be changed at any time at Microsofts discretion |
+| ApprovedForAllApps |  Any RSC enabled app can be consented to by users in your tenant. With these settings enabled, preapproval of RSC permissions is not needed as all RSC enabled apps are approved for use |
+| ApprovedForPreApprovedApps |  RSC enabled apps can be consented on an app-by-app basis. Admins can choose which apps are approved for which RSC permissions. |
```

Comment:
> ApprovedForPreApprovedApps |  RSC enabled apps can be consented on an app-by- [](http://example.com/codeflow?start=1&length=78)

We shouldnt add ApprovedForPreApprovedApps in the documentation right now. It will be added once we're ready to announce preapprovals. @AkJo to confirm.